akka-stream/src/main/scala/akka/stream/javadsl/Graph.scala

```diff
@@ -278,10 +277,10 @@ object FlowGraph {
    * The [[Builder]] is mutable and not thread-safe,
    * thus you should construct your Graph and then share the constructed immutable [[FlowGraph]].
    */
-  def builder(): Builder = new Builder()(new scaladsl.FlowGraph.Builder)
+  def builder[Mat](): Builder[Mat] = new Builder()(new scaladsl.FlowGraph.Builder[Mat])
```

Comment:
So, what do you propose? AFAICS we are completely free to change this identifier to anything we want, but `MaterializedType` is out due to obesity.